cfme/tests/services/test_service_customer_bz.py

```diff
@@ -67,3 +71,46 @@ def test_edit_bundle_entry_point(appliance, provider, catalog_item, request):
                                                               "ServiceProvision_Template/"
                                                               "CatalogItemInitialization")
     view.cancel_button.click()
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[BZ(1705021)])
+@pytest.mark.parametrize(
+    "import_data",
+    [DatastoreImport("bz_1705021.zip", "bz_1705021", None)],
+    ids=["sample_domain"],
+)
+@pytest.mark.parametrize("file_name", ["bz_1705021.yml"],
```

Comment:
May want to in-line comment here that you're parametrizing this for import_dialog fixture.